composer.json

```diff
@@ -22,7 +22,11 @@
     "require": {
         "php": ">= 5.6",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "~6.0"
+        "php-http/httplug": "^2.0",
```

Comment:
`^1.0 || ^2.0` I'd say, as 2.0 is limited to php 7 (or 7.1, can't remember). The "interface" stays the same though (the signatures stays the same, there was only a typehint added IRC)

Reply:
Thanks a lot for raising this point! I was actually planning to look into this 😄As you mention, 2.0 only supports PHP 7 and above. It would be great to flexibilize it so people can still use this with an outdated PHP version - even though I don't think we should build for that. Anyway, I also saw a lot of client implementations aren't compatible with 2.0 yet, so that also adds up for making this happen. I'll give it a try.

Reply:
AFAIC, php 5 should rot now (even 7.0). :D But just adding support for 1.0 (through `^1.0 || ^2.0`) should do the trick, so that when clients supporting 2.0 will be made, you'll have nothing to do. :)

Reply:
Yeah makes sense! According to HTTPPlug changelog, as you mentioned there's no breaking change, it's only a change in the method signatures for the client implementations. I just gave it a try allowing both versions and installed the [php-http/curl-client](https://github.com/php-http/curl-client) library (which is still not upgraded to HTTPPlug 2 and works fine, so we can go ahead this way 🎉